code/messaging/src/main/java/com/adobe/marketing/mobile/aepuitemplates/utils/Constants.kt

```diff
@@ -0,0 +1,98 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepuitemplates.utils
+
+/**
+ * Object containing constant values used throughout the application.
+ *
+ * This class is structured to hold constants related to card templates, including schema data,
+ * UI elements, and specific styling details. These constants are referenced in various UI model
+ * classes to ensure consistency and avoid hardcoding string values.
+ */
+class Constants {
```

Comment:
Can we class this `AepUITemplateConstants` to avoid confusion? `Constants` is a very generic file name